community/ceph/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Contributor: John Coyle <dx9err@gmail.com>
 # Maintainer: John Coyle <dx9err@gmail.com>
 pkgname=ceph
-pkgver=14.2.0
-pkgrel=3
+pkgver=14.2.1
+pkgrel=0
 pkgdesc="Ceph is a distributed object store and file system"
 pkgusers="ceph"
 pkggroups="ceph"
```

Comment:
LGPL-2.1
GPL-2.0

are missing the `-or-later` or `-only` suffix for SPDX compliance

use `AND` instead of `and`

use `WITH` keyword with the name exception to denote an exception to the license

e.g: `GPL-2.0-or-later WITH Autoconf-exception-2.0`

Reply:
Should I do this in this PR or a follow up PR?

Reply:
in this PR. squash it and git push --force

Reply:
Done. Sorry it took so long.